nullaway/src/main/java/com/uber/nullaway/handlers/RxNullabilityPropagator.java

```diff
@@ -148,22 +148,23 @@
    */
 
   // Set of filter methods found thus far (e.g. A.filter, see above)
-  private final Set<MethodTree> filterMethodsSet = new LinkedHashSet<>();
+  private Set<Tree> filterMethodOrLambdaSet = new LinkedHashSet<Tree>();
```

Comment:
why not `final` anymore?

Reply:
No real reason, bad merge, will fix. Thanks.